src/controllers/bulk/bulkcontroller.h

```diff
@@ -56,6 +56,7 @@ class BulkController : public Controller {
 
     virtual bool savePreset(const QString fileName) const;
 
+    virtual void visit(const KeyboardControllerPreset* preset);
```

Comment:
Sure, `ControllerPresetVisitor`- and `ConstControllerPresetVisitor`'s `visit` functions are now renamed to...
- `visitKeyboard`
- `visitMidi`
- `visitHid`

... just as you proposed. It's indeed more clear now, nice.